cmd/subctl/cloud.go

```diff
@@ -69,6 +72,8 @@ func init() {
 	cloudPrepareCmd.PersistentFlags().Var(&uint16Slice{value: &ports}, "metrics-ports", "Metrics ports")
 	_ = cloudPrepareCmd.PersistentFlags().MarkDeprecated("metrics-ports", "Metrics ports are no longer used, this flag is ignored")
 
+	cloudPrepareCmd.PersistentFlags().BoolVar(&cloudOptions.useLoadBalancer, "load-balancer", false, "deploy using LoadBalancer service")
```

Comment:
```suggestion
	cloudPrepareCmd.PersistentFlags().BoolVar(&cloudOptions.useLoadBalancer, "use-load-balancer", false, "Deploy using the load balancer service")
```

Reply:
I think we should be consistent with join command [1], it makes sense to keep flag name as  'load-balancer' 
I'll update the description 


[1]

https://github.com/submariner-io/subctl/blob/devel/cmd/subctl/join.go#L86

Reply:
OK but we should just reuse the code between the  two instead of duplicating:

```Go
func addLoadBalancerFlag(cmd *cobra.Command, p *bool) {
    cmd.Flags().BoolVar(p, "load-balancer", false, "enable automatic LoadBalancer in front of the gateways")
}
```